### PR TITLE
Added typing information for initiator

### PR DESCRIPTION
**Pull request by hackaugusto**

### CODE REVIEW

File: raiden/utils/typing.py [hunk 1]
```diff
@@ -1,10 +1,30 @@
 # -*- coding: utf-8 -*-
-from typing import *  # NOQA
+from typing import *  # NOQA pylint:disable=wildcard-import,unused-wildcard-import
+from typing import NewType
 
-Address = bytes
-Secret = bytes
-Signature = bytes
-Keccak256 = bytes
+_Address = bytes
```
[the remark this answers is not in the document]
Comment (LefterisJP) [reply raiden/utils/typing.py]: So let me understand. The `_Type` is a normal type while `Type` is a type that can be used for runtime checks due to the `NewType()` call?

Isn't this a bit ugly? I don't know much about python typing so I am just investigating here. Do we need to have two kinds for each type?

**Comment by LefterisJP** [comment]
There were some conflicts. Rebased on master.

File: raiden/utils/typing.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (LefterisJP) [reply raiden/utils/typing.py]: So I am okay with this PR but perhaps let's just call it different. `T_TypeName`?

**Comment by palango** [comment]
I fixed an error and renamed to `T_TypeName`

File: raiden/utils/typing.py [hunk 2]
```diff
@@ -1,10 +1,30 @@
...
+BlockExpiration = NewType('BlockExpiration', int)
+
+T_BlockNumber = int
+BlockNumber = NewType('BlockNumber', int)
+
+T_BlockTimeout = int
+BlockTimeout = NewType('BlockNumber', int)
+
+T_ChannelID = T_Address
+ChannelID = NewType('ChannelID', Address)
```
Comment (hackaugusto) [inline raiden/utils/typing.py]: This is wrong.

`NewType` returns a function, so `Address` is a function.

> However, at runtime, NewType('Derived', Base) will return a dummy function that simply returns its argument

And the docs says:

> The second argument must be a properly subclassable class, i.e., not a type construct like Union, etc.

which `function` is not.

https://mypy.readthedocs.io/en/latest/kinds_of_types.html#newtypes

**Pull request merged**